compiler/README.md

```diff
@@ -353,26 +345,20 @@ ownership of `y` is transfered to `z`. This means that the memory associated
 with `y` will never be freed, but the `AbortCompilation` will be caught and
 execution will continue, and this memory will have leaked.
 
-It's best to just never use `unique_ptr::release` in sorbet_llvm.
+It's best to just never use `unique_ptr::release` in the compiler.
 
 
 ## Editor and environment
 
-Nearly all of the [Editor and
-environment](https://github.com/sorbet/sorbet#editor-and-environment) tips from
-the Sorbet repo apply here too.
+Nearly all of the [Editor and environment] tips from the Sorbet repo apply here
+too.
+
+[Editor and environment]: https://github.com/sorbet/sorbet#editor-and-environment
```

Comment:
Do we need this?  If we do, it seems weird to talk about the tips from the same repo that we're in?